guides/common/assembly_enabling-and-synchronizing-the-project-client-name-repository.adoc

```diff
@@ -0,0 +1,9 @@
+include::modules/con_enabling-and-synchronizing-the-project-client-name-repository.adoc[]
+
+include::modules/proc_enabling-the-project-client-name-repository-for-rhel-9-and-rhel-8.adoc[leveloffset=+1]
```

Comment:
I'd flip the order: sync (for Satellite) first; enable (on hosts) second

Reply:
Applied.